### ISSUE TITLE
There is an error running this command, is it a LOKI-OXEN upgrade problem

### ISSUE THREAD

**Issue by TIDealHub** [jagerman/loki-multi-sn#1]
hello
There is an error running this command, is it a LOKI-OXEN upgrade problem, please check, thank you.

sudo **oxen**-multi-sn-create 42
Creating /etc/oxen/node-42.conf
Creating /etc/oxen/storage-42.conf
Creating /etc/loki/lokinet-router-42.ini
[NFO] lokinet 2021-02-10 10:39:14.908 GMT [+0.000s] config/config.cpp:1144 Attempting to create config file for router at "/tmp/lokinet.PQGWwo2PNS/lokinet.ini"
[NFO] lokinet 2021-02-10 10:39:14.908 GMT [+0.000s] config/config.cpp:1161 Generated new config "/tmp/lokinet.PQGWwo2PNS/lokinet.ini"
downloading https://seed.lokinet.org/lokinet.signed
lokinet successfully bootstrapped

Done with configuration setup. Enabling services...
Failed to enable unit: Unit file **loki**-node@42.service does not exist.

**Comment by jagerman** [comment]
Thanks, should be fixed in the 0.19 version of the package I just uploaded.

You may need to use:

```
sudo systemctl enable --now oxen{-node,-storage-server,net-router}@NN.service
```
(replacing NN with the number you used) for the existing service you created to start it up.  (Drop `--now` if you want to enable it for automatic startup without actually starting right away).

**Comment by TIDealHub** [comment]
Thank you very much, it works fine already!
